src/main.go

```diff
@@ -10,10 +10,8 @@ import (
 )
 
 func setMetadata(name string, value string) (string, error) {
-	return "nothing", nil
-
-	//out, err := exec.Command("buildkite-agent", "meta-data", "set", name, value).CombinedOutput()
-	//return string(out), err
+	out, err := exec.Command("buildkite-agent", "meta-data", "set", name, value).CombinedOutput()
```

Comment:
Ahh good catch. I remember we commented this out for testing, having it here makes rebuilds work really well